src/utils/søknad.ts

```diff
@@ -59,3 +60,38 @@ export const fraStringTilTall = (tallAvTypenStreng: string) => {
   }
   return parsed;
 };
+
+export const verdiTilTekstsvar = (verdi: string | Date | boolean) => {
+  if (typeof verdi === 'string') {
+    return verdi;
+  } else if (typeof verdi === 'boolean') {
+    if (verdi === true) {
+      return 'Ja';
+    } else {
+      return 'Nei';
+    }
+  } else if (verdi instanceof Date) {
+    return formatDate(verdi);
+  } else {
+    return null;
+  }
+};
+
```

Comment:
Smooth loope funksjon for oppsummering! 😄 
I stedet for hardkodet "Ja" og "Nei" svar, så kunne man returnet `hentTekst('svar.ja', intl)` og tilsvarende for hvis verdi: boolean er false.  😸

Reply:
Og i noen tilfeller vil verdi være en string[]. F.eks. i Aktivitet og DinSituasjon vil det første spm være et fleralternativspørsmål. Hvordan håndteres strenglister her? 🤔

Reply:
Dette er kanskje oppsummeringsspesifikt tho 🙈

Reply:
Bytta til å hente tekstene 😊 Godt poeng det med string[]. Jeg noterer det på Favro-kortet og håndterer det når jeg kommer dit.